ansible/roles-infra/infra-gcp-template-destroy/tasks/main.yml

```diff
@@ -48,9 +48,26 @@
 
 - name: Destroy method deployment-manager
   block:
+    - name: Delete the project secrets
+      command: "gcloud secrets delete {{ project_tag }}-bastion-ssh-secret -q"
+      environment:
+        PATH: '{{ output_dir }}/google-cloud-sdk/bin:/usr/bin:/usr/local/bin'
+        CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: "{{ gcp_credentials_file }}"
+        CLOUDSDK_COMPUTE_REGION: "{{ gcp_region }}"
+        CLOUDSDK_CONFIG: "{{ output_dir }}/.gcloud-{{ guid }}"
+        CLOUDSDK_CORE_PROJECT: "{{ gcp_project_id }}"
+      ignore_errors: yes
```

Comment:
Why this ignore_errors is needed ?
Can it be replaced by a combination of `changed_when` and `failed_when` ?

Reply:
I'll try that out